x-pack/plugins/infra/server/plugin.ts

```diff
@@ -31,6 +31,7 @@ import { LogEntriesService } from './services/log_entries';
 import { InfraPluginRequestHandlerContext } from './types';
 import { UsageCollector } from './usage/usage_collector';
 import { createGetLogQueryFields } from './services/log_queries/get_log_query_fields';
+import { handleEsError } from '../../../../src/plugins/es_ui_shared/server';
```

Comment:
Do you know if there is any difference in how things build based on whether this import uses `../..` or just `from 'src/plugins/etc` ? I've seen both and I have no idea if there is any real difference.

Reply:
I don't, but wondering the same thing.  I'll ask.